docs/attributes-registry/db.md

```diff
@@ -26,13 +26,15 @@
 | `db.instance.id` | string | An identifier (address, unique name, or any other identifier) of the database instance that is executing queries or mutations on the current connection. This is useful in cases where the database is running in a clustered environment and the instrumentation is able to record the node executing the query. The client may obtain this value in databases like MySQL using queries like `select @@hostname`. | `mysql-e26b99z.example.com` | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
 | `db.name` | string | This attribute is used to report the name of the database being accessed. For commands that switch the database, this should be set to the target database (even if the command fails). [1] | `customers`; `main` | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
 | `db.operation` | string | The name of the operation being executed, e.g. the [MongoDB command name](https://docs.mongodb.com/manual/reference/command/#database-operations) such as `findAndModify`, or the SQL keyword. [2] | `findAndModify`; `HMSET`; `SELECT` | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
-| `db.statement` | string | The database statement being executed. | `SELECT * FROM wuser_table`; `SET mykey "WuValue"` | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| `db.query.parameter.<key>` | string | The query parameters, `<key>` being the parameter name, the value being the parameter value.
```

Comment:
bringing up discussion from the sig meeting:
- the parameter can be used more than once (`select * from foo where bar != "baz" and bar != "something else"`)
- the operator could be interesting and value can be complicated (`select * from foo where bar not in ["baz","something else"])

Reply:
To my understanding parameters are for the placeholders (named or indexed) in a prepared query.
If the placeholder is used several times it's still identified by e.g.  `$1` or `:id` - so given once as input.

I don't think we should parse anything from a `where` clause in the statement and convert it into a parameter.

Reply:
> bringing up discussion from the sig meeting:
> 
> * the parameter can be used more than once (`select * from foo where bar != "baz" and bar != "something else"`)
> * the operator could be interesting and value can be complicated (`select * from foo where bar not in ["baz","something else"]`)

I think this is related to the discussion above about what parameters are.

my thinking is that parameters are only for placeholder values, so

> `select * from foo where bar != "baz" and bar != "something else"`

wouldn't have any parameters, while:

> `select * from foo where bar != ? and bar != ?

would have two parameters `db.query.parameter.0` and `db.query.parameter.1`

and

> `select * from foo where bar != :baz and bar != :other

would have two parameters `db.query.parameter.baz` and `db.query.parameter.other`